### PR TITLE
Add new Tips and Tricks blogpost

### PR DESCRIPTION
**Pull request by sw-yx**
## What does this PR do?

Add new Tips and Tricks blogpost

### CODE REVIEW

File: blog/2021-07-30-temporal-tips-tricks-1.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+author: swyx
+author_title: Head of Developer Experience
+author_image_url: https://avatars.githubusercontent.com/u/6764957?v=4
+release_version: V1.11.2
+---
+
+
+<!--truncate-->
+
+Discussions about Temporal often get into very high minded distributed systems debates, but sometimes it can be helpful to swap small tips that can have a huge impact on developer productivity!
```
Comment (flossypurse) [inline blog/2021-07-30-temporal-tips-tricks-1.md:18]:
```suggestion
Discussions about Temporal often get into very high-minded distributed systems debates, but sometimes it can be helpful to swap small tips that can have a huge impact on developer productivity!
```

Comment (flossypurse) [reply blog/2021-07-30-temporal-tips-tricks-1.md:18]: otherwise sounds like your mind is "high" ;)

File: blog/2021-07-30-temporal-tips-tricks-1.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+You can export the event history via `[tctl](https://docs.temporal.io/docs/system-tools/tctl#show-workflow-history)` or the [Web UI](https://docs.temporal.io/docs/system-tools/web-ui#execution-histories-on-temporal-web):
+
+```bash
+tctl workflow show \
+  -w myWorkflowId \
+  -r 866ae14c-88cf-4f1e-980f-571e031d71b0
+```
+
+This is not just good for recordkeeping, but also great for migrating Workflows to new versions *while* they are still running!
```
Comment (rylandg) [inline blog/2021-07-30-temporal-tips-tricks-1.md:95]: What does this have to do with new versions?

File: blog/2021-07-30-temporal-tips-tricks-1.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+import { ResponsivePlayer } from '../src/components'
+
+<ResponsivePlayer url='https://www.youtube.com/watch?v=kkP899WxgzY' />
+
+Before a migration, you can [replay event histories in a test](https://github.com/tsurdilo/temporal-versioning-go/blob/033e04a75049c4bea79908e0619332360c3cddc3/tests/replay_test.go), to verify that you have written backward compatible code:
```
Comment (rylandg) [inline blog/2021-07-30-temporal-tips-tricks-1.md:101]: I think this is not the core value. The core value is that you can take a failing production workflow and debug it locally exactly as it was executed.

**Review by rylandg — APPROVED** [review]
Small comments, overall LGTM

File: blog/2021-07-30-temporal-tips-tricks-1.md (hunk 2 above)
Comment (sw-yx) [reply blog/2021-07-30-temporal-tips-tricks-1.md:95]:
```suggestion
This is not just good for recordkeeping, but also great for taking a failing production workflow and debug it locally, exactly as it was executed.
```
